core/operation/src/main/java/uk/gov/gchq/gaffer/operation/impl/output/ToCsv.java

```diff
@@ -38,26 +39,28 @@
  *
  * @see ToCsv.Builder
  */
-@JsonPropertyOrder(value = {"class", "input", "elementGenerator"}, alphabetic = true)
+@JsonPropertyOrder(value = {"class", "input", "csvGenerator"}, alphabetic = true)
 @Since("1.0.0")
 @Summary("Converts elements to CSV Strings")
 public class ToCsv implements
         InputOutput<Iterable<? extends Element>, Iterable<? extends String>>,
         MultiInput<Element> {
 
     @Required
-    private CsvGenerator elementGenerator;
+    private CsvGenerator csvGenerator;
     private Iterable<? extends Element> input;
     private boolean includeHeader = true;
     private Map<String, String> options;
 
+    private CsvFormat csvFormat;
+
     @JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, include = JsonTypeInfo.As.PROPERTY, property = "class")
-    public CsvGenerator getElementGenerator() {
-        return elementGenerator;
+    public CsvGenerator getCsvGenerator() {
+        return csvGenerator;
     }
 
-    void setElementGenerator(final CsvGenerator elementGenerator) {
-        this.elementGenerator = elementGenerator;
+    void setCsvGenerator(final CsvGenerator csvGenerator) {
```

Comment:
Sure this isn't better as public? Default is package private? Looks like it could be private without any consequences because the only use is in this class. Although I question the reason for a setter if this method is not used in any other class.

Reply:
In fact, as this is used in the builder, it should be public like the other methods which are only used in the builder.